huangjoyce3/MealMatch
# Calendar backend

### Changes

- filled in backend functionality for calendar
  - listener on `delivery_indices` and each delivery for the active month
  - gets refreshed when navigating between months
- some dope UI improvements

### Testing
- log in with all 3 account types and make various edits
- check future vs past deliveries

### TODOs
don't think there are any??

![image](https://user-images.githubusercontent.com/6366341/39110264-58bcc2a4-4685-11e8-817d-84bcc7d2f4a4.png)
